src/XRP/wallet-factory.ts

```diff
@@ -1,22 +1,30 @@
-import Wallet from './wallet'
 import Utils from '../Common/utils'
 
+import Wallet from './wallet'
+
 /**
  * Encapsulates various methods for generating Wallets.
  */
 export default class WalletFactory {
+  public readonly isTest: boolean
```

Comment:
We disallow parameter properties on classes, because being more explicit is easier for people new to TypeScript (and ES6 classes don't have parameter properties).